.travis.yml

```diff
@@ -1,19 +1,18 @@
-# Sample .travis.yml for R projects
-
 language: r
-sudo: false
+sudo: required
 cache: packages
 
 r_binary_packages:
-  - lme4
+  - rstanarm
+  - rstan
   - dplyr
-  - purrr
-  - devtools
-  - git2r
-  - data.table
-  - stringi
-  - matrixStats
-  - StanHeaders
+  - Matrix
```

Comment:
Matrix is a recommended package, so there is no binary, you will need to remove this

Reply:
I would try with *just* `-rstan` and `-rstanarm`